dali/operators/coord/coord_flip.cc

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/coord/coord_flip.h"
+
+namespace dali {
+
+DALI_SCHEMA(CoordFlip)
+    .DocStr(
+        R"code(Transforms coordinates so that they are flipped (point reflected) with respect
+to a center point.)code")
+    .NumInput(1)
+    .NumOutput(1)
+    .AddOptionalArg<TensorLayout>(
+      "layout",
+      R"code(Determines the layout of the coordinates. Possible values are:
+
+  - ``x`` (horizontal position), 
+  - ``y`` (vertical position), 
+  - ``z`` (depthwise position),
+
+Note: If left empty, ``"x"``, ``"xy"`` or ``"xyz"`` will be assumed, depending on the number of dimensions.
+)code",
+      TensorLayout{""})
+    .AddOptionalArg("horizontal", R"code(Flip horizontal dimension.)code", 1, true)
```

Comment:
maybe they should be called flip_x, flip_y, flip_z ? Just a thought/question. Current ones are compatible with other operators, but since the order of coordinates is different here (XYZ vs (Z)YX as in (D)HWC layout), this compatibility may cause more problems than it solves.

Reply:
done